controllers/Templates.php

```diff
@@ -52,6 +53,39 @@ private function getIndex()
         return ($final);
     }
 
+    function getXMLFromValidation($url, $paramsFromGet)
+    {
+
+        $url .= '?' . http_build_query($paramsFromGet);
+
+        $ch = curl_init();
+        curl_setopt($ch, CURLOPT_URL, $url);
+        curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+
+        $response = curl_exec($ch);
+
+        // Handle any errors
+        if (curl_errno($ch)) {
+            echo 'Curl error: ' . curl_error($ch);
+            curl_close($ch);
+            return null;
+        }
+
+        curl_close($ch);
+
+        //Loads the XML response into a SimpleXMLElement object
+        $xml = simplexml_load_string($response);
+        if ($xml === false) {
+            echo "Failed loading XML\n";
```

Comment:
This should not output errors

Reply:
fixed, handled outisde of the method, in index file